pages/apis/rest_api/builds.md

```diff
@@ -338,6 +338,8 @@ curl -H "Authorization: Bearer $TOKEN" \
 > 📘 Webhook URL
 > The response only includes a webhook URL in `pipeline.provider.webhook_url` if the user has edit permissions for the pipeline. Otherwise, the field returns with an empty string.
 
+The `state` of a build will not return `blocked`, as is the case with [notifications](/docs/pipelines/notifications#build-states). Instead, it will return `true` for the value of `blocked`.
```

Comment:
Thanks @mcncl for this contribution!

Two things pushed up directly:

- Remove the callout formatting as the web renderer doesn't support 2 consecutive callouts together.
- Start a new sentence from 'Instead...'.

Query, what is 'it' that returns a value of `true`, since `state` doesn't appear to have a valid value of `true`?

Using the word 'it' for the subject of a sentence is inherently ambiguous, unless it relates directly to the object of the previous sentence or clause, when the meaning of 'it' in such contexts is more obvious.